extensions/wikia/ThemeDesigner/ThemeDesignerHelper.class.php

```diff
@@ -4,13 +4,15 @@ class ThemeDesignerHelper {
 
 	public static function checkAccess() {
 		global $wgUser;
+
 		// @FIXME when we're out of beta editinterface needs to be removed and themedesgner set to true for sysops
 		return $wgUser->isAllowed( 'themedesigner' );
 	}
 
-	public static function parseText($text = "") {
+	public static function parseText( $text = "" ) {
 		global $wgTitle;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)